openbas-front/src/components/common/pagination/PaginationComponent.tsx

```diff
@@ -101,7 +102,7 @@ const PaginationComponent = <T extends object>({
   // Filters
   const [openMitreFilter, setOpenMitreFilter] = React.useState(false);
 
-  useEffect(() => {
+  const fetchData = () => {
```

Comment:
Why this change ?

If fetchData is not called by any other component.

Reply:
Forgetting to go back during a test